source/extensions/filters/http/jwt_authn/jwt_cache.cc

```diff
@@ -56,7 +60,10 @@ class JwtCacheImpl : public JwtCache {
   }
 
   void insert(const std::string& token, std::unique_ptr<::google::jwt_verify::Jwt>&& jwt) override {
-    if (jwt_lru_cache_ && token.size() <= kMaxJwtSizeForCache) {
+    if (!jwt_lru_cache_ || token.size() > std::numeric_limits<unsigned int>::max()) {
```

Comment:
```suggestion
    if (!jwt_lru_cache_ || token.size() > std::numeric_limits<uint32_t>::max()) {
```

FWIW, I don't think the right-hand-side can be triggered, as AFAIK `size()` return type is size_t which should map to the same type.

Reply:
Sorry for that leftover, I fixed it.

> I don't think the right-hand-side can be triggered, as AFAIK size() return type is size_t

Thanks for this hint. Actually, this check is unrealistic. Theoretically `token.size()` could be bigger than the max value of `int32_t`, but I do not believe that it may ever happen - a token would have to be bigger than 4Gb (4294967295 bytes). Do you think this check makes sense? Or is it better to assume that it won't happen and check `static_cast<uint32_t>(token.size()) > max_token_size`?